stdlib/csv/0/csv.rbs

```diff
@@ -1722,7 +1722,10 @@ class CSV < Object
   #     would read `UTF-32BE` data from the file but transcode it to `UTF-8`
   #     before parsing.
   #
-  def self.foreach: [U] (String | IO | StringIO path, ?::Hash[Symbol, U] options) { (::Array[String?] arg0) -> void } -> void
+  def self.foreach: (String | IO path, ?String mode, headers: true, **untyped options) { (::CSV::Row arg0) -> void } -> void
```

Comment:
Accurately, the `headers` keyword argument can't be only received true, but also truthy value, for example, String and Symbol, etc.

However, As headers argument is often provided true, I defined true only in this methods.
Please tell me the way of definitions much better than this if you know. Thanks.